.github/workflows/run-client.yml

```diff
@@ -0,0 +1,67 @@
+name: run client
+on:
+  push:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  pipeline-user-scenario:
+    name: Build on node ${{ matrix.node_version }} and ${{ matrix.os }}
```

Comment:
This needs to be changed.

Reply:
All of jobs of workflow is complete, what should be changed here ?